maquetado/scss/styles.scss

```diff
@@ -34,6 +34,16 @@
   width: 100%;
 }
 
+.navbar {
+  height: 100%;
+  max-height: 70px;
+  overflow: hidden;
+  padding: 0 155px;
```

Comment:
why this padding?

Reply:
To have both the Wolox logo image and Logout label in the places where are described in the invision template

Reply:
what if your screen is larger? does it break?